backend/sozisel/lib/sozisel/model/quizzes/quiz_question.ex

```diff
@@ -0,0 +1,26 @@
+defmodule Sozisel.Model.Quizzes.QuizQuestion do
+  use Sozisel.Model.Schema
+  import Ecto.Changeset
+
+  @type t :: %__MODULE__{
+          question: String.t(),
+          answers: [String],
+          correct_answers: [Integer]
```

Comment:
I guess it means indexes? What if we decided to shuffle questions? Checking correct answers then would be a pain.